src/test/resources/hudson/tasks/junit/JENKINS-41134/TestSuite_second.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+    <testsuite name="TestSuite_second" tests="1" errors="0" failures="1" skip="0" timestamp="2012-04-05T10:50:00">

```

Comment:
This exact scenario you described above appears to have bitten us. We have mocha tests that run extraordinarily fast, and sure enough we have three different test suites with the same name, same timestamp, but one of them has failures. Jenkins was only occasionally reporting the failures. Guessing it was just the luck of the draw as to which one it picked to represent the pass/fail state. 

Is there an issue created to represent this scenario?